### ISSUE TITLE
React 17 Dependency

### ISSUE THREAD

**Issue by fergusstrange** [perawallet/connect#22]
The React 17 dependency is causing issues when using a build of react that is newer. During `npm install --save @perawallet/connect` the following error is displayed. Is it possible to make this a peer dependency or upgrade to match the later React versions?

```
npm ERR! Found: react@18.0.0
npm ERR! node_modules/react
npm ERR!   react@"18.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17.0.2" from @perawallet/connect@0.1.4
npm ERR! node_modules/@perawallet/connect
npm ERR!   @perawallet/connect@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by mucahit** [comment]
Hey @fergusstrange, thanks for the feedback!

We're on it to upgrade `react` peer dependency to `18.x`

**Comment by mucahit** [comment]
Hey @fergusstrange, it's released with `0.2.0`. https://github.com/perawallet/connect/releases/tag/0.2.0

Thank you!
